### ISSUE TITLE
Imports from sibling directory not parsed using babel config

### ISSUE THREAD

**Issue by mikestopcontinues** [babel/babel-eslint#825]
I'm porting my codebase to a monorepo, but I've found when importing code from sibling directories, the syntax isn't parsed correctly. In particular, the pipeline operator, which I assume isn't included in whatever fallback is being used.

My dir structure:

```
/
- appOne/
  - package.json w/ babel and eslint config
  - js that imports
- common/
  - js that's imported
```

When I run eslint in `appOne`, all of my custom syntax within `appOne` tests fine, but the code imported from common explodes.

If babel-eslint@11 is really using my babel config, it should work. After all, my code compiles fine.

Here's my config:
```js
{
  "babel": {
    "presets": ["babel-preset-contour"], // proposals, presets, nothing weird.
    "ignore": ["node_modules"],
    "plugins": [["module-resolver", { // admittedly, weird.
      "cwd": "packagejson",
      "root": ["./", "../common"]
    }]]
  },
  "eslintConfig": {
    "extends": ["eslint-config-contour"], // includes "parser": "babel-eslint" v11.0.0-beta.2
    "settings": {
      "import/resolver": { // to match above resolver — I know it's not likely to factor in
        "node": {
          "extensions": [".js", ".jsx"],
          "paths": ["./", "../common"]
        }
      }
    }
  }
}
```

Finally, I do get this error, right at the start of linting:
```
`parseForESLint` from parser `<ROOT>/node_modules/babel-eslint/lib/index.js` is invalid and will just be ignored
```

What can I try? What's going on here?

**Comment by kaicataldo** [comment]
To circle back, we discussed this on Slack and it sounded like @mikestopcontinues was able to resolve this. If that's the case, do you mind closing this issue?

**Comment by mikestopcontinues** [comment]
Absolutely! Thanks again!
